gulpfile.js

```diff
@@ -122,37 +122,24 @@ gulp.task('build:backend:sass:generate-css', function () {
     .pipe(livereload())
 })
 
-gulp.task('build:backend', function () {
-  gulp.start(
-    'build:backend:assets:copy-css-vendors',
-    'build:backend:assets:copy-fonts-vendors',
-    'build:backend:assets:copy-js-vendors',
-    'build:backend:assets:copy-fine-uploader-css-and-images',
-    'build:backend:sass:generate-css',
-    'build:backend:assets:copy-ckeditor'
-  )
-})
+const buildBackend = gulp.parallel(
+  'build:backend:assets:copy-css-vendors',
+  'build:backend:assets:copy-fonts-vendors',
+  'build:backend:assets:copy-js-vendors',
+  'build:backend:assets:copy-fine-uploader-css-and-images',
+  'build:backend:sass:generate-css',
+  'build:backend:assets:copy-ckeditor'
+)
 
 gulp.task('serve:backend', function () {
   livereload.listen()
   gulp.watch(
-    [
-      'src/Backend/Core/Layout/Sass/**/*.scss'
-    ],
-    ['build:backend:sass:generate-css']
+    'src/Backend/Core/Layout/Sass/**/*.scss',
+    gulp.parallel('build:backend:sass:generate-css')
```

Comment:
Dit snap ik niet, is toch maar 1 taak?

Reply:
Klopt, maar alle tasks moeten in Gulp 4 aangeroepen worden in `gulp.series()` of `gulp.parallel()`. Zonder werkt dus niet.

Reply:
Aha ok, dus al er meerdere taken zijn moet er beslist worden of ze in parallel of in serie moeten.

Reply:
Klopt.